rules/es6.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
 	"rules": {
+		// DISABLED UNTIL MORE FLEXIBLE:
 		// Require braces around arrow body when required
-		"arrow-body-style": [2, "as-needed"],
+		//"arrow-body-style": [2, "as-needed"],
```

Comment:
Let's not comment any rules out, but instead just turn them off--so the 2 becomes a 0. My rationale for this isn't fantastic, but commenting it out seems hacky when a disabling option already inherently exists.